### ISSUE TITLE
Import is reserved keyword

### ISSUE THREAD

**Issue by VitaliyR** [apexskier/nova-eslint#282]
Hey!

Trying to use plugin - gives me that `import is reserved` in the Nova, but running eslint separately on any file works fine - without errors.

**To reproduce**

1. Open any file with `import` declaration

Or

1. Create new TS file
2. Add import declaration - no errors
3. Save it - after save - error appears

**Expected behavior**

Should follow the output of the eslint run from cli

**Screenshots**

<img width="452" alt="image" src="https://user-images.githubusercontent.com/2149612/159982859-5a94e1c7-be2c-4e93-b635-03848c0ed31e.png">

**Environment**

- Nova version: 9.1 383286
- Extension version: 1.7.2
- ESLint version: v7.18.0
- macOS version: 12.3 (21E230)
- NodeJS information:
  - node version: v14.19.0
  - npm version: 6.14.16
  - installation method: dmg file

**Extension console output**

```
ESLint[20:10:27.512000] activating...
ESLint[20:10:27.691000] activated
```

**Comment by VitaliyR** [comment]
Found the issue.

Broken in #278 - `src/process.ts:223`

As my file path became really wrong for `stdin-filename` argument -> eslint just gets crazy

Expected path: absolute or relative, real e.g. app/src/file.ts (absolute /Users/user/Work/app/src/file.ts)
Got: /Work/app/src/file.ts

**Comment by kopischke** [comment]
**EDIT:** ~~Looks like one contract change in the Nova API that is left standing after the 9.0 to 9.1 kerfuffle is that `decodeURI()` now removes the `file://` prefix. That results in #278 not actually reverting to a working state, because the URI massaging step now erroneously removes two directories too many.~~

I was wrong about that – whatever the cause, it is not `decodeURI()` removing the prefix, because it doesn’t (did it do that before Nova 9? The code in 1.7.0 almost leads me to believe it did, but as I have sown enough confusion already in #277, I will refrain from further wild claims).

**Comment by ahcurrier** [comment]
Got it! Thanks for the update.

**Comment by drwpow** [comment]
FWIW I’m a big fan of using [URL](https://nodejs.org/api/url.html#urlfileurltopathurl) for Node.js paths. It does a very good job of resolving filesystem paths for both unix and Windows (and I can help with a PR if this is helpful)

**Comment by apexskier** [comment]
> FWIW I’m a big fan of using [URL](https://nodejs.org/api/url.html#urlfileurltopathurl) for Node.js paths. It does a very good job of resolving filesystem paths for both unix and Windows (and I can help with a PR if this is helpful)

@drwpow Unfortunately, extensions don't run in a Node.js runtime. They run in a customized JavaScriptCore runtime with Nova's standard library. That being said, if you try it out and it works a PR would be great!

**Comment by apexskier** [comment]
Included in release [v1.7.3](https://github.com/apexskier/nova-eslint/releases/tag/v1.7.3)
